lib/src/transaction/council_node/node_join_transaction_builder.ts

```diff
@@ -128,6 +129,17 @@ export class NodeJoinTransactionBuilder extends TransactionBuilder {
         return this;
     }
 
+    /**
+     * Returns unsigned raw transaction in hex
+     *
+     * @throws {Error} error when transaction is not completed
+     * @returns {Buffer}
+     * @memberof NodeJoinTransactionBuilder
+     */
+    public toUnsignedHex(): Buffer {
+        return Buffer.concat([Buffer.from('0102', 'hex'), this.unsignedRawTx]);
```

Comment:
can '0102' be put in a global variable, or is it supposed to be this hard constant across all network chain IDs?

Reply:
`0102` and the similar parts are the hardcoded bytes in our encoding. Yes, this is a good suggestion, I can refactor to put all the encoding prefixes into a single file and import them in the builders.

Reply:
Refactored to `lib/src/transaction/encoding.ts`